src/externals/scheduler.js

```diff
@@ -1,9 +1,26 @@
 const axios = require('axios').default
+const { GATEWAY_ADDRESS } = process.env
 
 module.exports = class Scheduler {
   constructor() {
     this.axios = axios.create({
       timeout: 1000
```

Comment:
thêm giúp t 1 option này
```suggestion
      timeout: 1000,
      validateStatus: null,
```